### DISCUSSION TITLE
Markdown : CPE書式を使うと絵文字が出てしまう

### DISCUSSION THREAD

**Discussion by witchcraze** [increments/qiita-discussions#184] [ヘルプ] [answered]
仕方のないことなのかもしれませんが
`cpe:2.3:a:oracle:mysql:8.0.0:*:*:*:*:*:*:*` を普通に使うと
cpe:2.3:a:oracle:mysql:8.0.0:*:*:*:*:*:*:* となってしまいます。

絵文字への変換を制御することは可能でしょうか？

**Comment by kskwtnk** [comment, marked as answer]
@witchcraze 
ご意見ありがとうございます！

QiitaのパーサーがGitHub Flavored Markdown準拠でやっているのもあり、現状GitHubでも絵文字になっていることも含めて、制御の対応は行わないでおけたら……と思っています
現状書いていただいているように、バッククオートで括って記載していただけますと幸いです 🙏

**Reply by witchcraze** [reply]
回答ありがとうございます。
了解いたしました。
